dask_kubernetes/operator/controller/tests/test_controller.py

```diff
@@ -227,6 +224,48 @@ async def test_simplecluster(k8s_cluster, kopf_runner, gen_cluster):
             )
             assert _EXPECTED_ANNOTATIONS.items() <= worker_annotations.items()
 
+            # Assert labels from the dask cluster are propagated into the dask worker group
+            workergroup_labels = json.loads(
+                k8s_cluster.kubectl(
+                    "get",
+                    "daskworkergroups",
+                    "--selector=dask.org/component=workergroup",
+                    "-o",
+                    "jsonpath='{.items[0].metadata.labels}'",
+                )[1:-1]
+            )
+            for label in _EXPECTED_LABELS:
+                assert label in workergroup_labels
+                assert workergroup_labels[label] == _EXPECTED_LABELS[label]
```

Comment:
This can be simplified to
```suggestion
            assert _EXPECTED_LABELS.items() <= workergroup_labels.items()
```

Reply:
Same for the others

Reply:
Thanks @jacobtomlinson , fixed up

Reply:
tests have just passed :)